test/framework/alertmanager.go

```
@@ -257,19 +254,22 @@ func (f *Framework) WaitForAlertmanagerInitialized(ns, name string, amountPeers
 	return nil
 }
 
-func (f *Framework) GetAlertmanagerStatus(ns, n string) (models.AlertmanagerStatus, error) {
+func (f *Framework) GetAlertmanagerStatus(ns, n string) models.AlertmanagerStatus {
```

`GetAlertmanagerStatus()` shouldn't swallow the error.